<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1042</width>
    <height>607</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>20</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,8,1">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QPushButton" name="btn_open">
     <property name="text">
      <string>Open Image</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lbl_image">
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../datasets/cat_dog/test/cat_test01.jpg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lbl_result">
     <property name="text">
      <string>Cat  or Dog? </string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btn_open</sender>
   <signal>clicked()</signal>
   <receiver>lbl_image</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>286</x>
     <y>266</y>
    </hint>
    <hint type="destinationlabel">
     <x>184</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
